lib/publishing_api_payload.rb

```diff
@@ -87,13 +87,17 @@ def details
     details = {
       political: edition.political?,
       change_history: history.change_history,
+      attachments: attachments,
     }
 
     if document_type.lead_image? && edition.lead_image_revision.present?
       details[:image] = image
     end
 
-    details[:documents] = [] if document_type.attachments.featured?
+    if document_type.attachments.featured?
+      details[:documents] = []
+      details[:featured_attachments] = featured_attachments
```

Comment:
I don't think we need to do this as part of this story. I've just been talking to @barrucadu about the usage of this field (as I was a bit surprised by our usage) and I think we're going to change the approach in content schemas.